src/utilities/handleArmySubmission.js

```diff
@@ -0,0 +1,33 @@
+import firebase from 'firebase/app';
+import store from '../Redux/store';
+import 'firebase/firestore';
+
+export default function handleArmySubmission(input) {
```

Comment:
I'd personally break up the logic in the if methods into private non exported methods.